DVR/SessionDataTask.swift

```diff
@@ -35,7 +35,12 @@ class SessionDataTask: NSURLSessionDataTask {
         // Find interaction
         if let interaction = cassette?.interactionForRequest(request) {
             // Forward completion
-            completion?(interaction.responseData, interaction.response, nil)
+            if let completion = completion {
+                let queue = dispatch_queue_create("com.venmo.DVR.sessionDataTaskQueue", nil)
```

Comment:
Should we create a new queue for every data task? Would it may be better to have just one static queue on which these tasks run?